systemd/service_checks.json

```diff
@@ -0,0 +1,11 @@
+[
+    {
+        "agent_version": "6.9.0",
```

Comment:
We'll want to update this when you're ready 🙂

Reply:
Updated to `6.11.0`